docs/gettingstarted/quickstart.ipynb

```diff
@@ -12,12 +12,22 @@
    "cell_type": "markdown",
```

Comment:
<p>Can we have this be <code>%pip install lf-tape</code></p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/lincc-frameworks/tape/pull/414/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>